go/client/cmd_simplefs_copy.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"errors"
+	"os"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+// CmdSimpleFSCopy is the 'simplefs list' command.
+type CmdSimpleFSCopy struct {
+	libkb.Contextified
+	src     keybase1.Path
+	dest    keybase1.Path
+	recurse bool
+}
+
+// NewCmdSimpleFSCopy creates a new cli.Command.
+func NewCmdSimpleFSCopy(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "cp",
+		ArgumentHelp: "<source> [dest]",
```

Comment:
We still might need support on Windows though!

To test it out, run this program via `node`

```
console.log(process.argv.slice(2));
```

Note that if `*foo*` fails to deglob, your app will wind up with the argument list `[ "*foo*"]`